fbc/4.12/openshift-builds-operator/catalog.yaml

```
@@ -52,249 +64,377 @@ name: openshift-builds-1.0
 package: openshift-builds-operator
 schema: olm.channel
 ---
+schema: olm.deprecations
+package: openshift-builds-operator
+entries:
+  - reference:
+      schema: olm.bundle
+      name: openshift-builds-operator.v1.0.0
+    message: |
+      openshift-builds-operator.v1.0.0 is deprecated. Uninstall and install openshift-builds-operator.v1.1.0 or later.
```

This is more of an "end of life" notice for the product version. We should be asking customers to upgrade, not uninstall/reinstall. @siamaksade is this okay language?

```suggestion
      openshift-builds-operator.v1.0.0 has reached the end of maintenance support. Please upgrade to openshift-builds-operator.v1.1.0 or later.
```